src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -116,17 +118,18 @@ export const View = (props: Props) => {
   const tabSpecificExposures = getTabSpecificExposures(selectedTab);
   const filteredExposures = filterExposures(tabSpecificExposures, filters);
   const exposureCardElems = filteredExposures.map((exposure: Exposure) => {
-    const exposureCardKey = `${isScanResult(exposure) ? "scan" : "breach"}-${
-      exposure.id
-    }`;
+    const exposureCardKey = isScanResult(exposure)
+      ? "scan-" + exposure.onerep_scan_result_id
```

Comment:
Is `id` not enough to distinguish the different OneRep exposures?

Reply:
Is `exposure.id` the scan ID, and `exposure.onerep_scan_result_id` the scan result ID?

If so then I think `id` isn't enough, if the goal is to have a unique ID for each of the exposure cards, I think scan result ID (or something derived from it) is probably what's needed.

Reply:
Yeah, it should be enough _combined with the other changes in this PR_, specifically [this one](https://github.com/mozilla/blurts-server/pull/3760/files#diff-2f5dac2b258572f0c86ba000438f9fd3cb9fd22320519cbb189df216979ee854R53), which ensures that the ID is now actually the ID of our database row of the scan result, rather than the ID of the database row of the scan.

However, I figured it might be a good idea to do some defense-in-depth: `onerep_scan_result_id` should _also_ be able to distinguish between different exposures, and has the additional advantage of being specific to scan results, and thus avoiding the risk of accidentally referring to a different value (like happens in the current code on `main`).

And since OneRep is the source of truth anyway, it also feels better to rely on their identifiers than on the ones of our DB cache? But I don't have a strong argument for that feeling, in terms of what could go wrong otherwise :)